src/heplers/assert.ts

```diff
@@ -0,0 +1,49 @@
+import { ErrorRsp } from '../generated/index.js';
```

Comment:
@sam-corbado Rename folder from heplers to helpers :)

Reply:
Good catch! Done. 👍